tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -0,0 +1,147 @@
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="9B", name='CHILD_RECIPIENTS', type='number', startIndex=159, endIndex=167,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="10B", name='NONCUSTODIALS', type='number', startIndex=183, endIndex=191,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="11B", name='AMT_ASSISTANCE', type='number', startIndex=211, endIndex=223,
+              required=True, validators=[validators.isInLimits(0, 999999999999)]),
+        Field(item="12B", name='CLOSED_CASES', type='number', startIndex=243, endIndex=251,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+    ],
+)
+
+s3 = RowSchema(
+    model=SSP_M6,
+    preparsing_validators=[
+        validators.hasLength(259),
+    ],
+    postparsing_validators=[
+        validators.sumIsEqual(
+            "SSPMOE_FAMILIES", ["NUM_2_PARENTS", "NUM_1_PARENTS", "NUM_NO_PARENTS"]),
+        validators.sumIsEqual(
+            "NUM_RECIPIENTS", ["ADULT_RECIPIENTS", "CHILD_RECIPIENTS"]),
+    ],
+    fields=[
+        Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+              required=True, validators=[]),
+        Field(item="2", name='CALENDAR_QUARTER', type='number', startIndex=2, endIndex=7,
+              required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                         validators.quarterIsValid()]),
+        TransformField(calendar_quarter_to_rpt_month_year(2), item="2B", name='RPT_MONTH_YEAR', type='number',
+                       startIndex=2, endIndex=7, required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                                                            validators.dateMonthIsValid()]),
+        Field(item="3C", name='SSPMOE_FAMILIES', type='number', startIndex=23, endIndex=31,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="4C", name='NUM_2_PARENTS', type='number', startIndex=47, endIndex=55,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="5C", name='NUM_1_PARENTS', type='number', startIndex=71, endIndex=79,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="6C", name='NUM_NO_PARENTS', type='number', startIndex=95, endIndex=103,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="7C", name='NUM_RECIPIENTS', type='number', startIndex=119, endIndex=127,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="8C", name='ADULT_RECIPIENTS', type='number', startIndex=143, endIndex=151,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
```

Comment:
Is this supposed to have a validator or did Alex make a mistake in the spreadsheet?